dashboard/src/components/Code.js

```diff
@@ -19,16 +20,28 @@ const styles = theme => ({
         ? theme.palette.secondary.light
         : theme.palette.secondary.dark,
   },
+  rightMargin: {
+    whiteSpace: "pre-wrap",
+    marginRight: "24px",
+  },
 });
 
 class Code extends React.PureComponent {
   static propTypes = {
     classes: PropTypes.object.isRequired,
+    dictionaryMargin: PropTypes.bool,
```

Comment:
I probably confused you with my comment before but I think this may be a bit too specific.